crates/cli-support/src/js/binding.rs

```diff
@@ -982,11 +1029,7 @@ fn instruction(
             js.push(format!("!isLikeNone({0})", val));
             js.push(format!(
                 "isLikeNone({val}) ? {zero} : {val}",
-                zero = if *ty == ValType::I64 {
-                    "BigInt(0)"
-                } else {
-                    "0"
-                }
+                zero = if *ty == ValType::I64 { "0n" } else { "0" }
```

Comment:
This might regress on #3164. We may want to avoid bigint literals everywhere else too.

Reply:
Good catch. Thank you! This is definitely a blocker.

After looking at #3164 and the error it intended to fix, I now believe that #3164 shouldn't have been accepted. I'll open an issue about where I'll explain my reasoning in detail.

Reply:
I'll just use `BigInt` constructors for now.